src/decoding.cc

```diff
@@ -1,10 +1,11 @@
 #include "ctranslate2/decoding.h"
-
 #include <cmath>
 #include <limits>
 #include <map>
 
 #include "ctranslate2/ops/ops.h"
+#include "ctranslate2/ops/min_max.h"
+#include "ctranslate2/ops/log.h"
```

Comment:
Move these includes inside `ctranslate2/ops/ops.h` so that you don't need to add them here.